### PR TITLE
Add help method for key vault to create keyVault keyStore

### PR DESCRIPTION
**Pull request by zhichengliu12581**
1. make extensive way to set up the keystore as a helper method. 
2. Fix the NullPointerException that may cause when the certificateBundle is empty

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultLoadStoreParameter.java [hunk 1]
```diff
@@ -74,6 +74,22 @@ public KeyVaultLoadStoreParameter(String uri, String tenantId, String clientId,
...
+    /**
+     * Constructor.
+     *
+     * @param uri the Azure Key Vault URI.
+     * @param tenantId the tenant ID.
+     * @param clientId the client ID.
+     * @param clientSecret the client secret.
+     * @param managedIdentity the managedIdentity.
+     */
+    public KeyVaultLoadStoreParameter(String uri, String tenantId, String clientId, String clientSecret, String managedIdentity) {
```
Comment (chenrujun) [inline sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultLoadStoreParameter.java:86]: Could you please make other constructor call this one?

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultLoadStoreParameter.java (hunk 1 above)
Comment (chenrujun) [reply sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultLoadStoreParameter.java:82]: There are still 2 constructors not calling this constructor, could you please update it?

Comment (zhichengliu12581) [reply sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultLoadStoreParameter.java:78]: My fault, I forgot this, it has been added in the latest commit.

**Review by chenrujun — APPROVED** [review]
LGTM.

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**